srcpkgs/acr/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'acr'
 pkgname=acr
-version=1.7.2
+version=1.9.3
 revision=1
 archs=noarch
 build_style=gnu-configure
 short_desc="AutoConf Replacement"
-maintainer="criw <contact@thecriw.com>"
```

Comment:
Are you the current maintainer? Why is it orphaned?

Reply:
Package has missed updates for almost 2 years, the maintainer hasn't been active

Reply:
Considered adoption?

Reply:
@fnogcps mind adopting?

Reply:
Yeah, no problem.